examples/Huggingface_Transformers/Pipeline_handler_generalized.py

```diff
@@ -0,0 +1,178 @@
+'''
+TODO:
+(i) Add support for remaining Pipeline tasks
+(ii) Add support for Captum explanations
+
+'''
+
+import os
+import torch
+import base64
+import io
+from PIL import Image
+
+from ts.torch_handler.base_handler import BaseHandler
+from transformers import pipeline
+
+
+#Reference: https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.pipeline.task
+pipeline_supported_tasks= [
```

Comment:
Well yes actually, if we totally forget the supported things in `Transformer_handler_generalized.py` like `FasterTransformer`, `do_lower_case`, `save_mode`, `model_parallel` etc whose integration didn't seem really straightforward at that time, I believe that literally all the tasks in the supported pipeline could be ported to be deployable to Torchserve.  

As I mentioned earlier, the integration of remaining tasks is left to maintainers of the repo and it can be done in a step-by-step manner for these tasks one at a time:
* See how Image-classification and Text-classification is supported in the code of `Pipeline_handler_generalized.py`. 
* Read documentation of that task in [pipeline docs](https://huggingface.co/docs/transformers/main_classes/pipelines) to figure out what kind of input it expects (example: For text-classification it expects a list of text, for image-classification it expects either list of image paths or list of PIL Image instances).  
* Implement an appropriate function in `HFPreprocessDispatcher` to deserialize inputs and output a format expected by the pipeline as discovered in last step, can copy code from default handlers (except Audio does not have a Default Handler so I'm clueless, but there must be some way to preprocess it)
* Implement postprocessing if necessary in `HFPostprocessDispatcher`.  

I could have implemented more functions but I'm needed elsewhere.

Reply:
I'm getting a bit lost as to what's in scope for this PR vs what's in future plans 

It seems like this PR includes
1. A pipeline handler that works for sentiment analysis and image classification
2. Helper scripts that create a mar file
3. Some documentation to run the above with Docker

I ask because I can't merge future plans only what exists right now, I like that we can do something new like image classification and your pipeline pre/post process dispatchers, I'm still not sure I follow why we need to create `create_hf_handler.sh`, why an extra `config.properties` is included and why the changes couldn't be made to the existing handler. I do agree with the goal that if we use HF pipelines that it'll help us more easily support new models and take advantage of optimium optimizations but I'm not sure this is the right implementation. There are some important features that haven't been figured out either like request batching which for this PR to be used need to be done.

If you don't have time to answer these questions now, keep your PR open and we can revisit when someone on the team has more time to contribute code.